config.sh

```diff
@@ -69,13 +85,34 @@ function pre_build {
         build_freetype
     fi
 
+    if [ -z "$IS_OSX" ]; then
+        export FREETYPE_LIBS=-lfreetype
+        export FREETYPE_CFLAGS=-I/usr/local/include/freetype2/
+    fi
+    build_simple harfbuzz $HARFBUZZ_VERSION https://github.com/harfbuzz/harfbuzz/releases/download/$HARFBUZZ_VERSION tar.xz --with-freetype=yes --with-glib=no
+    if [ -z "$IS_OSX" ]; then
+        export FREETYPE_LIBS=''
+        export FREETYPE_CFLAGS=''
+    fi
+
     # Append licenses
     for filename in dependency_licenses/*; do
       echo -e "\n\n----\n\n$(basename $filename | cut -f 1 -d '.')\n" | cat >> Pillow/LICENSE
       cat $filename >> Pillow/LICENSE
     done
 }
 
+function pip_wheel_cmd {
+    local abs_wheelhouse=$1
+    if [ -z "$IS_OSX" ]; then
+        CFLAGS="$CFLAGS --std=c99"  # for Raqm
```

Comment:
I think it is better to compile with C99 than to merge in the (poorly tested) C89 patch from https://github.com/python-pillow/Pillow/pull/5062/commits/b3cfe73854e74bc25a88f53b177713bfb63812e4.